common/types/store-types.ts

```diff
@@ -112,6 +112,14 @@ export type Group = {
   readonly classCode: string;
 };
 
+/**
+ * The type of a group member.
+ */
+export type GroupMember = {
+  readonly netId: string;
+  readonly name: string;
```

Comment:
We might also want a field for photo URL

Reply:
It's probably good for now. This PR just use hardcoded data, and we will naturally switch to use the existing `SamwiseUserProfile` once we integrate with real data.